build/mathquill.d.ts

```diff
@@ -22,6 +22,47 @@ declare namespace MathQuill {
       handlers?: HandlerOptions;
     };
 
+    interface MQNode {
+      id: number;
+      parent: NodeBase;
+    }
+  
+    interface MQSelection {
+      id: number;
+      getEnd(dir: number): number;
+    }
+  
+    interface NodeBase extends MQNode {
```

Comment:
`extends MQNode` does not feel right to me here, but it's the only thing that worked with the rest of the types in Perseus.